service-binding/postgresql-crunchy/src/test/java/io/quarkus/ts/sb/postgresql/OpenShiftPostgreSqlSbIT.java

```diff
@@ -0,0 +1,73 @@
+package io.quarkus.ts.sb.postgresql;
+
+import java.nio.file.Paths;
+
+import javax.inject.Inject;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.bootstrap.inject.OpenShiftClient;
+import io.quarkus.test.scenarios.OpenShiftDeploymentStrategy;
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.quarkus.test.utils.Command;
+
+@OpenShiftScenario(deployment = OpenShiftDeploymentStrategy.UsingOpenShiftExtensionAndDockerBuildStrategy)
+public class OpenShiftPostgreSqlSbIT {
+
+    @Inject
+    static OpenShiftClient ocClient;
+
+    @QuarkusApplication
+    static RestService app = new RestService()
+            .onPreStart(s -> createPostgresCluster());;
+
+    @AfterAll
+    public static void deletePostgresCluster() {
```

Comment:
And afterAll static method could have a more "generic name" something like "tearDown" or ..."afterAll" so if in the future we want to add more statements into this method, we don't have to change the method name.

Reply:
Updated.